policy/modules/system/logging.te

```diff
@@ -548,6 +548,16 @@ ifdef(`init_systemd',`
 	systemd_manage_journal_files(syslogd_t)
 
 	udev_read_runtime_files(syslogd_t)
+
+	# needed for systemd --user instances
+	dontaudit syslogd_t self:capability dac_read_search;
+
+	userdom_search_user_runtime_root(syslogd_t)
+	userdom_search_user_runtime(syslogd_t)
+
+	systemd_search_user_runtime(syslogd_t)
+	systemd_read_user_runtime_files(syslogd_t)
```

Comment:
afaik  it only needs to traverse and read lnk files (ie /run/user/*/systemd/units/*)

Reply:
OK. If I allow `dac_read_search` and dontaudit these, it should be OK. I will test it.